api/response/schema_query.py

```diff
@@ -88,38 +86,37 @@ def get_all_reponses(self, info):
         rooms = RoomModel.query.all()
         for room in rooms:
             room_name = room.name
-            total_response = ResponseModel.query.filter_by(
-                room_id=room.id).count()
             room_response = Response.get_query(info).filter_by(room_id=room.id)
+            total_response = room_response.count()
             all_responses = Query.get_room_response(self, room_response)
             responses = RoomResponse(
                 room_name=room_name,
```

Comment:
Thanks @bevkololi, but which line are you referring to. If it is line 90. I believe it is required as the total response is passed in the RoomResponse

Reply:
Line 93.

Reply:
Okay. That line is needed @bevkololi . This is because room_name is part of the data returned inside a room response. This is different from the room_name returned outside the responses.
If you look at my screenshots in the PR description, you may notice the difference.